scripts/run_presubmit_checks.py

```diff
@@ -81,8 +88,12 @@ def main(args=None) -> None:
 
     print('Comparing the current branch with %s' % branch)
 
-    all_changed_files = subprocess.check_output([
-        'git', 'diff', '--cached', '--name-only', '--diff-filter=ACM', branch])
+    all_changed_files = subprocess.check_output(
+        [
+        'git', 'diff', '--cached', '--name-only', '--diff-filter=ACM', branch
```

Comment:
Ditto

Reply:
The reason is mentioned in this comment https://github.com/oppia/oppia/pull/16226#discussion_r991137346

Also, the indentation is fixed!